src/App.js

```diff
@@ -18,16 +18,21 @@ function App() {
 
 
   function addTodo (newTodo) {
-    setTodoList([...todoList, newTodo])
+    setTodoList((prevList) => [...prevList, newTodo])
+  }
+
+
+  function removeTodo (todo) {
+    const newArray = todoList.filter((list) => todo.id !== list.id )
+    setTodoList(newArray)
   }
```

Comment:
Good job using `filter` here 💯 

Remember like last time, though. If we're using the current state to calculaute/set the new state, we should add a function to the setter (`setTodoList`) to get the current state. React says its the safest way for some reason.

So it'd look something like this:
```suggestion
  function removeTodo (todo) {
    setTodoList((currentTodoList) => currentTodoList.filter((list) => todo.id !== list.id ) )
  }
```